### ISSUE TITLE
The filename and the fallback cannot contain the "/" and "\" characters.

### ISSUE THREAD

**Issue by dathwa** [OFFLINE-GmbH/oc-mall-plugin#1101]
Hello.
Client created a virtual product and set the dispaly name with a slash - e.g. "Fun/Games".

When customer tried to download after purchase, the following error was thrown.

```
The filename and the fallback cannot contain the "/" and "\" characters.
Class InvalidArgumentException
/public_html/vendor/symfony/http-foundation/HeaderUtils.php
```
Any chance the display name can be "cleaned" before this happens?

Thank-you

**Comment by SamBrishes** [comment]
Hello,

thanks for your bug report. I fixed this issue in [commit 70bf2ba](https://github.com/OFFLINE-GmbH/oc-mall-plugin/commit/70bf2ba5888a631ae5cbb42dde3c9ea10eae4e32). Will be available in v3.2.0.

Sincerely,\
Sam

**Comment by SamBrishes** [comment]
This issue has been fixed and released with v3.2.0 / v3.2.1.
